massgov/mayflower-patternlab
# DP-9243: MF fix info details to show contact and related in sidebar as well as bottom on desktop

## Description
update css to show sidebar content at correct screen widths

## Related Issue / Ticket

- [DP-9243](https://jira.mass.gov/browse/DP-9243)

## Steps to Test

1. View the Information details page at /?p=pages-information-details
On desktop you should see the contact and link list in the sidebar  

## Screenshots
<img width="1512" alt="screen shot 2018-06-11 at 11 01 40 am" src="https://user-images.githubusercontent.com/18662734/41243068-e01aab46-6d66-11e8-8b07-ef91439289c7.png">
